src/picknik_ur_mock_hw_config/objectives/move_along_path.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Move along Path">
+  <!-- ////////// -->
+  <BehaviorTree ID="Move along Path" _description="Moves the robot tip on a rectangular path." _favorite="true" _hardcoded="false">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Action ID="MoveToWaypoint" waypoint_name="Look at Right Table" planning_group_name="manipulator" controller_names="/joint_trajectory_controller" use_all_planners="false" constraints=""/>
```

Comment:
Should we be concerned about people changing/moving this waypoint?

Reply:
This should still work even if that waypoint changes a bit. But of course it won't work if it's a very different waypoint.
We could use `CreateJointState` + `MoveToJointState` instead if you think that's better. But it would be slightly less portable to other robots. With the current UI it is easier to visualize/create a named waypoint than it is to create a custom joint state message. Also it looks like those predefined waypoints are used in other behaviors as well? But let me know what you prefer, happy to go one way or the other.